couchdbkit/schema/base.py

```diff
@@ -154,7 +154,7 @@ def save_docs(cls, docs, use_uuids=True, all_or_nothing=False):
         db = cls.get_db()
         if any(doc._doc_type != cls._doc_type for doc in docs):
             raise ValueError("one of your documents does not have the correct type")
-        db.bulk_save(docs, use_uuids=use_uuids, all_or_nothing=all_or_nothing)
+        db.bulk_save(tuple(docs), use_uuids=use_uuids, all_or_nothing=all_or_nothing)
```

Comment:
Could be nice to do this [here](https://github.com/dimagi/couchdbkit/blob/54a11541906a4dfa22253bae30fe8c2f4d41a907/couchdbkit/client.py#L544) since it looks like `db.save_docs` needs to have a list, and this could quite plausibly come up again elsewhere.